DataLayer/SongLibrary/DownloadSongs.sh

```diff
@@ -3,3 +3,11 @@
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-06+Animal.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-11+Armageddon+It.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-14+Hysteria.mp3
+sudo apt-get install eyeD3
```

Comment:
eyeD3 is a python library, so to install it you need to run

`pip install eyeD3`

sudo apt-get install won't work for this library.

Reply:
I was able to get the script to work by downloading eyeD3 from their website and not through command line